### PR TITLE
Removing ImplicitMSAL

### PR DESCRIPTION
**Pull request by nikithauc**
This will have merge conflicts with #484 . To be merged after #484.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,135 +1,132 @@
...
-		"uglify-es": "^3.3.9"
-	},
-	"peerDependenciesMeta": {
-		"@azure/identity": {
-			"optional": true
-		},
-		"buffer": {
-			"optional": true
-		},
-		"msal": {
```
Comment (baywet) [inline package.json:125]: there's an msal entry line 106

File: src/authentication/msal/MSALAuthenticationProviderOptions.ts [hunk 2]
```diff
@@ -10,27 +10,3 @@
  */
 
 import { AuthenticationProviderOptions } from "../../IAuthenticationProviderOptions";
-
```
Comment (baywet) [inline src/authentication/msal/MSALAuthenticationProviderOptions.ts:13]: why are we keeping the file at all?

**Review by baywet — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (nikithauc) [reply package.json:126]: Removed this. Thanks for pointing!

File: src/authentication/msal/MSALAuthenticationProviderOptions.ts (hunk 2 above)
Comment (nikithauc) [reply src/authentication/msal/MSALAuthenticationProviderOptions.ts]: Kept this file with MSALBrowserAuthProviderOptions.  This file will contain options related to MSAL library or libraries (TBD).

**Review by baywet — APPROVED** [review]

**Pull request merged**